docs/tools/easy_diff.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, sys
+import subprocess
+from git import cmd
+from tempfile import NamedTemporaryFile
+
+SCRIPT_PATH = os.path.abspath(__file__)
+CLICK_HOUSE_REPO_HOME = os.path.join(os.path.dirname(SCRIPT_PATH), '../../')
```

Comment:
not so important, but I think:
*  it's better to write `CLICKHOUSE` in one word
* `os.path.join(os.path.dirname(SCRIPT_PATH), '..', '..')` looks better